docs/DafnyRef/integration- rust/IntegrationRust.md

```diff
@@ -59,7 +59,7 @@ For maximum flexibility, Dafny follows these rules to encode types in Rust:
 | `int32`                       | `i32`                       |
 | `int16`                       | `i16`                       |
 | `int8`                        | `i8`                        |
-| `char`                        | `char` OR `u16`             |
+| `char`                        | `char` OR `u16` (`--unicode-chars=false`)  |
```

Comment:
I changed it to two rows instead and used the proper newtypes since they exist now. Newtypes in Rust make it possible to add methods that Dafny uses all the time.

|  Dafny type                   |   Rust type                 | Defined in       |
|-------------------------------|-----------------------------|------------------|
| `char`                        | `DafnyChar`                 | Runtime (`char`) |
| `char` (`--unicode-chars=false`)| `DafnyCharUTF16`          | Runtime (`u16`)  |